static/js/quality-chooser.js

```diff
@@ -44,7 +44,22 @@ $(document).ready(function() {
                   '&allowed=' + selectedAllowed +
                   '&preferred=' + selectedPreffered
         $.getJSON(url, function(data) {
-            $('#backlogged_episodes').text('Currently you have ' + data[1] + ' backlogged episodes. With this change you would have ' + data[0] + ' backlogged episodes');
+            var new_backlogged = parseInt(data[0])
```

Comment:
Change this, the data being passed back should already be an int if it's not then you need to fix the python.

After that it's fine